resource/preload/preload.go

```diff
@@ -36,13 +39,50 @@ const (
 )
 
 // Preload represents a preload link.
-type Preload interface {
-	// Header returns the value of Link HTTP header to perform the preload.
-	Header() string
-
-	// Resources returns a set of resources referenced by the preload link.
-	// It returns multiple resources when the preload offers more than one
-	// option, such as images with multi-source ("imagesrcset") or content
-	// negotiations ("variants").
-	Resources() []*resource.Resource
+type Preload struct {
+	*httplink.Link
+
+	// Resources contains a set of resources referenced by the preload link.
+	// It typically consists just of one resource, but contains multiple
```

Comment:
s/It/Resources maybe?

Reply:
I meant rather "the set of resources," so probably no. The meaning is the same whether "it" refers to Resources or the set of resources, so I leave it unchanged.